pcdsdevices/pseudopos.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+from ophyd.device import Component as Cpt
+from ophyd.pseudopos import PseudoPositioner, PseudoSingle
+
+
+class SyncAxesBase(PseudoPositioner):
+    """
+    Synchronized Axes.
+
+    This will move all axes to the same position at the same time. And report
+    its position as the average value of all axes.
+
+    Attributes
+    ----------
+    axes: ``list of str``
+        Names of the components that correspond to the real axes.
+    """
+    pseudo = Cpt(PseudoSingle)
+
+    def forward(self, pseudo_pos):
```

Comment:
This is a little oversimplified than what I think is needed for other applications. I think that is fine for now, but should be addressed later. For instance this will not preserve the difference between all the motors it will just set them to the same value. This will certainly cause issues with the stands but maybe the CCM has nice enough offsets that this doesn't matter

Reply:
CCM offsets are all zero. I didn't think we would be ok with having level stands with nonzero offsets.

Reply:
After some thought, probably best to record axis offsets before the first move and whenever requested and maintain those offsets with absolute moves